great_expectations/self_check/util.py

```diff
@@ -1318,7 +1319,7 @@ def get_test_validator_with_data(
                     type_ = schema[col]
                     if type_ in ["IntegerType", "LongType"]:
                         # Ints cannot be None...but None can be valid in Spark (as Null)
-                        vals = []  # type: ignore
+                        vals: list = []
```

Comment:
@Kilo59 Why not `List[str]`?

Reply:
`list` is the same as `List[Any]`

However I went ahead and updated it to `List[Union[str, int, float, None]]`